.env.example

```diff
@@ -2,3 +2,10 @@ NATIVE_RPC=http://localhost:26657
 NATIVE_GRPC=127.0.0.1:9090
 LOG_FILE_PATH="" # If empty, logs to STDOUT
 LOG_LEVEL="debug"
+SUI_CHAIN=http://127.0.0.1:9000
+SIGNER_ACCOUNT_MNEMONIC="defense frost latin party smoke veteran b"
+GAS_ADDRESS="0x50087bb2fc68c47e69e0b871e5ea5d0f0e9217117e5e1b167a91664c5c82"
+SMART_CONTRACT_ADDRESS="0xb1ec836a1c60581f884466b6ae76a0ec7987842443f56c125048ee97"
```

Comment:
We should remove values in those fields.

Reply:
Fixed